docs/DeveloperGuide.md

```diff
@@ -288,31 +288,100 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `BistroBoss` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Add an employee**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to add employee contact
+2.  BistroBoss requests contact information of employee
+3.  User provides required information
+4.  BistroBoss adds the employee contact
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 2a. The contact book is empty.
```

Comment:
I think the contact book being empty shouldn't be an error here, since we are adding a contact, not viewing a contact, I think can remove.

Reply:
done